config/ePO-default/xsdAndRdfDataTypes.xml

```diff
@@ -35,6 +36,8 @@
     <datatype namespace="rdf" qname="rdf:XMLLiteral"/>
     <datatype namespace="rdf" qname="rdf:langString"/>
     
-    <datatype namespace="rdfs" qname="rdfs:Literal"/>    
+    <datatype namespace="rdfs" qname="rdfs:Literal"/>   
+    
+    <datatype namespace="skos" qname="skos:Concept"/>
```

Comment:
I'm afraid ```skos:Concept``` is not a data type, and therefore we should not keep it in this list.